hack/logcheck.conf

```diff
@@ -14,7 +14,6 @@
 -structured .*
 
 # Now enable it again for migrated packages.
-structured k8s.io/kubernetes/cmd/kube-proxy/.*
```

Comment:
I think you have to move it to the "contextual" section below?

Reply:
Yes, I missed this, already added it.